src/ui/FacetValueSuggestions/FacetValueSuggestions.ts

```diff
@@ -34,7 +34,7 @@ export interface IQuerySuggestionKeyword {
 /**
  * This component provides [`Omnibox`]{@link Omnibox} query suggestions scoped to distinct categories based on the values of a specific [`field`]{@link FacetValueSuggestions.options.field}.
  *
- * See [Providing Facet Value Suggestions](https://docs.coveo.com/504/)
+ * See [Providing Search Box Suggestions](https://docs.coveo.com/340/)
```

Comment:
You can do:

@externaldocs [Providing Facet Value Suggestions](https://docs.coveo.com/en/340/#providing-facet-value-suggestions)